nixos/modules/services/networking/syncthing.nix

```diff
@@ -34,11 +34,9 @@ in
       };
 
       dataDir = mkOption {
-        default = "/var/lib/syncthing";
+        default = "~/.config/syncthing";
```

Comment:
Only if the service is configured to start as root. But yes, I think /var/lib/syncthing is ok too as a default :+1: